contracts/pm/mixins/interfaces/MContractRegistry.sol

```diff
@@ -6,6 +6,22 @@ import "../../../rounds/IRoundsManager.sol";
 
 
 contract MContractRegistry {
+    /**
+     * @dev Checks if the system is paused
+     * @notice Executes the 'whenSystemNotPaused' modifier 'MixinContractRegistry' inherits from 'Manager.sol'
```

Comment:
I think the notice here should actually be a part of the `@dev` tag since it is developer documentation. Basing this off of the [NatSpec tag docs](https://solidity.readthedocs.io/en/v0.5.11/natspec-format.html#tags).

Reply:
fixed in 752bec9